appveyor.yml

```diff
@@ -7,6 +7,9 @@ environment:
   # See: http://stackoverflow.com/a/13751649/163740
   CMD_IN_ENV: "cmd /E:ON /V:ON /C obvci_appveyor_python_build_env.cmd"
 
+  # Workaround for https://github.com/conda/conda-build/issues/636
+  PYTHONIOENCODING="UTF-8"
```

Comment:
That is better than installing Miniconda3 :wink:

I will changed #6 once you merge this.